lib/internal/repl/await.js

```diff
@@ -239,10 +240,13 @@ function processTopLevelAwait(src) {
     state.prepend(last, 'return (');
     state.append(last.expression, ')');
   }
-
+  const hoistedDeclarations = ArrayPrototypeJoin(state.hoistedDeclarationStatements, '');
+  if (hoistedDeclarations.length > 0){
+    wrappedArray = [];
+  }
   return (
     ArrayPrototypeJoin(state.hoistedDeclarationStatements, '') +
-    ArrayPrototypeJoin(wrappedArray, '')
+    'ArrayPrototypeJoin(wrappedArray, '')'
```

Comment:
This doesn't look quite right to me?

Reply:
Oops, wasn't supposed to go in.
Tests should be good now